### ISSUE TITLE
Setting/Overwriting Document Properties using Patcher

### ISSUE THREAD

**Issue by 0x1234567890** [dolanmiu/docx#2701]
Hello everyone,

I'm currently working on a project where I'm creating and editing documents using the docx.js framework. My goal is to set or overwrite specific document properties such as Title, Tags, and Comments.

I've heard that this can be done using Patchers, but I'm not sure how to implement this. Can someone provide an example or guide on how to set or modify these properties in a document using docx.js and Patchers?

Here are the specific properties I want to edit:

Title
Tags
Comments
If you have any code examples or links to helpful resources, I would greatly appreciate it.

![image](https://github.com/user-attachments/assets/f0fb2607-c3b6-4823-a384-c711e0c4cc7d)

Thank you in advance for your help!

Best regards,

**Comment by ghost** [comment]
I don't think this is possible, both from looking at the documentation and the code. All it can do is replace placeholder values found in an existing document with text or other docx elements.

I think it's misleading to advertise this library as an easy way to "modify" docx files when the library just provides a glorified search and replace (yes with the added benefit that you can replace the placeholders with rich text but it's still not a generic docx editor)
